pkg/agent/server/command/command_native.go

```diff
@@ -34,7 +34,12 @@ func NewCmd(u *osauth.User, shell, term, host string, command ...string) *exec.C
 		"SHELL=" + shell,
 		"SHELLHUB_HOST=" + host,
 	}
-	cmd.Dir = u.HomeDir
+
+	if _, err := os.Stat(u.HomeDir); err != nil {
+		cmd.Dir = "/"
```

Comment:
We must log so the user knows what is happening.

Reply:
Done!

This is an example of message sent:
```WARN[0006] failed to open user's home dir                dir=/home/test error="stat /home/test: no such file or directory"```